src/jvm/test/resources/expected_logs/thread_creation_representation_test.txt

```diff
@@ -0,0 +1,65 @@
+|                                                                                                                                                   |                                                                                                                                                                                            |           a ➜ 0 at ThreadCreationRepresentationTest.callMe(ThreadCreationRepresentationTest.kt:34)                                                                                         |                                                                                                                                                                                            |
+|                                                                                                                                                   |                                                                                                                                                                                            |           a = 1 at ThreadCreationRepresentationTest.callMe(ThreadCreationRepresentationTest.kt:34)                                                                                         |                                                                                                                                                                                            |
+|                                                                                                                                                   |                                                                                                                                                                                            |   result: void                                                                                                                                                                             |                                                                                                                                                                                            |
+|                                                                                                                                                   | run()                                                                                                                                                                                      |                                                                                                                                                                                            |                                                                                                                                                                                            |
```

Comment:
Can we raise an issue (and solve it) to get rid of these `run()` calls and the lambda invocation calls below?

Reply:
#536